src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    /* List of output messages shown to users */
+    // Greetings
+    private static final String MESSAGE_LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String MESSAGE_INTRO = "Hello from\n" + MESSAGE_LOGO;
+    private static final String MESSAGE_BODY = "Hello! I'm Duke\n"
+            + "What can I do for you?";
```

Comment:
May I suggest to put the line 13 and 14 into a single line or perhaps to ensure the intentation is around 4 spaces

Reply:
Makes sense. Thanks for your suggestion!